src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -80,12 +80,12 @@ Microsoft.CodeAnalysis.Options.PerLanguageOption<> (forwarded, contained in Micr
 Microsoft.CodeAnalysis.ProjectChanges.GetChangedDocuments(bool onlyGetDocumentsWithTextChanges) -> System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.DocumentId>
 Microsoft.CodeAnalysis.Workspace.OnOutputRefFilePathChanged(Microsoft.CodeAnalysis.ProjectId projectId, string outputFilePath) -> void
 override Microsoft.CodeAnalysis.FileTextLoader.LoadTextAndVersionAsync(Microsoft.CodeAnalysis.Workspace workspace, Microsoft.CodeAnalysis.DocumentId documentId, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<Microsoft.CodeAnalysis.TextAndVersion>
+override Microsoft.CodeAnalysis.Options.OptionKey.Equals(object obj) -> bool (forwarded, contained in Microsoft.CodeAnalysis)
```

Comment:
Consider reverting, as this is just adding a newline at the end of the file and moving existing options.